docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -610,62 +665,82 @@ Configurations, Producer Properties and Advanced Producer Configurations.
 ====== Consumer Properties
 
 .Consumer configurable properties of spring-cloud-azure-stream-binder-servicebus
-[cols="<,<,<", options="header"]
+[cols="<,<,<,<", options="header"]
 |===
-|Property | Type |Description
+|Property | Type |Default |Description
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.requeue-rejected
 |boolean
+|false
 |If the failed messages are routed to the DLQ.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.checkpoint-mode
 | CheckpointMode
+| RECORD
 | The checkpoint mode of checkpointing message. The supported modes are `MANUAL` and `RECORD`.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.max-concurrent-calls
 | Integer
+| 1
 | Max concurrent messages that the Service Bus processor client should process.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.max-concurrent-sessions
 |Integer
+|1
 | Maximum number of concurrent sessions to process at any given time.
 
-|*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.session-aware
+|*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.concurrency
```

Comment:
I think to put a deprecated property here is to some extent confusing. We only need introduce all the properties supported in 4.x versions. So could you move this part to the migration guide of [service bus binder properties](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/appendix.html#_from_azure_spring_cloud_stream_binder_servicebus_to_spring_cloud_azure_stream_binder_servicebus)? We need a table 18 to show the mapping relationship between the legacy propertis and the current one, which is similar with table 15.